sample-apps/generic_deepgrow/README.md

```diff
@@ -2,9 +2,77 @@
+  
+- Input image size
+  
+    > By default, this App is programmed to work on images of size (128, 192, 192) for DeepGrow 3D and (256, 256) for DeepGrow 2D. However, researchers can change this according to the GPU memory for
+    their own task. This can be changed in **[./main.py](./main.py)**. We do not recommend changing the size for DeepGrow 2D however the size for DeepGrow 3D should be adjusted depending upon the object of interest.
+ 
+- [Spatial and intensity transformation](https://docs.monai.io/en/latest/transforms.html) for pre and post processing
+  
+  > By default, this App comes with the following transforms to pre process the images and simulate clicks:
+  > - [LoadImaged](https://docs.monai.io/en/latest/_modules/monai/transforms/io/array.html#LoadImage) -> Loads both image and label from a dictionary
+  > - [AsChannelFirstd](https://docs.monai.io/en/latest/transforms.html#aschannelfirstd) -> Transforms both image and label to have the channel as first dimension
+  > - [Spacingd](https://docs.monai.io/en/latest/_modules/monai/transforms/spatial/dictionary.html#Spacingd) -> Resample image and label to a different image space,
+  > - [Orientationd](https://docs.monai.io/en/latest/_modules/monai/transforms/spatial/dictionary.html#Orientationd) -> Reorient image and labels,
+  > - [AddChanneld](https://docs.monai.io/en/latest/_modules/monai/transforms/utility/array.html#AddChannel) -> Add an extra channel to the image and label,
+  > - [SpatialCropForegroundd](https://docs.monai.io/en/latest/transforms.html#cropforegroundd) -> This transform randomly changes image intensity by a gamma value, 
+  > - [Resized](https://docs.monai.io/en/latest/_modules/monai/transforms/spatial/dictionary.html#Resized) -> Resample image to a different size,
+  > - [NormalizeIntensityd](https://docs.monai.io/en/latest/_modules/monai/transforms/intensity/dictionary.html#NormalizeIntensityd) -> Normalize intensity of the image,
+
+
+  Transformations used for the clicks simulation [(DeepGrow App)](https://docs.monai.io/en/latest/apps.html)
+  > 
+  > - [FindAllValidSlicesd](https://docs.monai.io/en/latest/_modules/monai/apps/deepgrow/transforms.html#FindAllValidSlicesd) -> This transform finds/lists all valid slices in the label. Label is assumed to be a 4D Volume with shape CDHW, where C=1,
+  > - [AddInitialSeedPointd](https://docs.monai.io/en/latest/_modules/monai/apps/deepgrow/transforms.html#AddInitialSeedPointd) -> This transform adds random guidance as initial seed point for a given label,
+  > - [AddGuidanceSignald](https://docs.monai.io/en/latest/_modules/monai/apps/deepgrow/transforms.html#AddGuidanceSignald) -> This transform adds guidance signal (foreground and background clicks) for input image, 
+  > - DiscardAddGuidanced -> This transform discards foreground and background tensors with a probability define as an argument. 
```

Comment:
I think this is particular to the DeepEdit App.

Reply:
Except for the last one, I think. Removed DiscardAddGuidanced